src/app/data-entries/dropdown/dropdown-menu/dropdown-menu.component.ts

```diff
@@ -17,17 +10,8 @@ import {
     templateUrl: "./dropdown-menu.component.html"
 })
 export class DropdownMenuComponent implements OnInit {
-    @ViewChild(IgxOverlayOutletDirective) public igxOverlayOutlet: IgxOverlayOutletDirective;
-    @ViewChild(IgxDropDownComponent) public menu: IgxDropDownComponent;
-
     public items: any[] = [];
-    public text;
-
-    private positionSettings = {
```

Comment:
Default settings in current sample are start point Right-Bottom and direction Left-Bottom. In updated sample we have Left-Bottom start point and Right-Bottom direction. Should we have same settings in old and updated sample?

Reply:
We had here NoOp scroll strategy and in the new sample it will be Absolute scroll strategy!

Reply:
Thanks for pointing that out! I'll pass modified overlay settings through the markup to illustrate how that is done. For the other samples, I think the scroll strategy is not really relevant and passing settings in each of the samples will result in clutter.